Makefile

```diff
@@ -1,6 +1,7 @@
 DOCKER_IMAGE_URI_PATH=566277102435.dkr.ecr.eu-west-2.amazonaws.com/congenica/psga-nonprod
 DOCKER_IMAGE_TAG=dev_latest
 SARS_COV_2=sars_cov_2
+DOCKER_IMAGE_NAME=sars-cov-2-pipeline
```

Comment:
Is it worth ensuring the name of the Docker image follows the repo name? If so, this should be `pipeline-sars-cov-2`.